amr-wind/wind_energy/actuator/FLLC.cpp

```diff
@@ -38,6 +39,45 @@ void FLLCInit(
     data.dr[0] = vs::mag(view.pos[1] - view.pos[0]) / 2.;
     data.dr[npts - 1] = vs::mag(view.pos[npts - 1] - view.pos[npts - 2]) / 2.;
 
+    namespace interp = ::amr_wind::interp;
+
+    // Non-uniform radial distribution variables
+    size_t idx = 0;
+    amrex::Real dr;
```

Comment:
Same thing here. Please give more descriptive names of the variables. It is hard to review `dr` `dr1` and `dr2` because I have to keep in my mind what they all mean. If you name the variables based on what they actually are you will need less comments and it will be easier to understand.

Comments are helpful, but if you have to annotate every line of code to explain what is going on, then it means the code is confusing, you have to read twice as much and do more work. 

1) Read comments
2) read code
3) check the code does what the comments say
4) confirm that is the right behavior 

versus

1) read the code
2) read the comments when they are there to clarify the code
2) confirm the behavior is right

This compounds over time and as the code base grows.

Reply:
I modified most variable names to account for this. 
Some variables are temporary and I was not sure what to name them. 
Let me know if you prefer another naming convention for specific variables like dr1 and dr2.